docs/README.md

```diff
@@ -0,0 +1,6 @@
+# mongo-swift-driver Docs
+
+The MongoDB Swift driver contains two modules:
+
+- [MongoSwift](../mongoswift/index.html), containing an asynchronous API and a BSON library
```

Comment:
Oh I just realized these may need to be updated too. Otherwise lgtm

Reply:
fixed